documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-configuration-database-history-properties.adoc

```diff
@@ -0,0 +1,98 @@
+You can specify the following sets of connector properties to control how {prodname} interacts with the database history topic:
+
+
+* xref:{context}-database-history-properties-for-configuring-connector-behavior[Database history connector configuration properties] that control how {prodname} processes events that it reads from the database history topic.
+* xref:{context}-pass-through-database-history-properties-for-configuring-producer-and-consumer-clients[Pass-through database history properties] that control how Kafka producer and consumer clients interact with the Kafka broker.
+* xref:{context}-pass-through-configuration-properties-for-database-drivers[Pass-through database driver properties] that control the behavior of the database driver.
```

Comment:
DB driver pass-through properties don't apply to the schema history. They apply to all the connectors. So this should be pulled up one level, to the same level as DB history properties, not be part of them.

Reply:
Ah, ok. I guess I do need to pull those into a separate _partial_ file. Downstream, I ran into build problems when I tried to insert heading levels into the partial, so I ended up using the simple `.` followed by no whitespace. But, of course, that means the 3 headings are all on the same level.

 OK, I'll take care of that. Thanks.